Cargo.lock

```diff
@@ -265,19 +271,49 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "2.33.4"
+version = "2.34.0"
```

Comment:
It bothers me (significantly) that we now have _two_ versions of clap in our dependency tree. I'd hope we only had 3.x. Could you look into why?

Reply:
Sure, I'll look again into it, miss attention.